### PR TITLE
Attachment issue when multiple images are uploaded

### PR DESCRIPTION
**Pull request by vishalnarkhede**
# Submit a pull request

## CLA

- [ ] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required).
- [ ] The code changes follow best practices
- [ ] Code changes are tested (add some information if not applicable)

## Description of the pull request

- E.g., lets say user attaches multiple images and multiple files (or other attachments) ... in this case, attachments are not shown in message. This issues has been fixed
- Rest are just `prettier` fixes

### CODE REVIEW

File: src/components/MessageSimple.js [hunk 1]
```diff
@@ -441,14 +439,18 @@ export class MessageSimple extends PureComponent {
 
             <div>
               {hasAttachment &&
-                images.length <= 1 &&
```
Comment (vishalnarkhede) [inline src/components/MessageSimple.js]: this condition will cause all the other attachments to be skipped.

File: src/components/Attachment.js [hunk 2]
```diff
@@ -166,9 +166,7 @@ export class Attachment extends PureComponent {
 
     return (
       <div
-        className={`str-chat__message-attachment str-chat__message-attachment--${type} str-chat__message-attachment--${
```
Comment (vishalnarkhede) [inline src/components/Attachment.js]: prettier fix

File: src/components/AttachmentActions.js [hunk 3]
```diff
@@ -28,9 +28,7 @@ export class AttachmentActions extends React.PureComponent {
           <span key={0}>{text}</span>
           {actions.map((action) => (
             <button
-              className={`str-chat__message-attachment-actions-button str-chat__message-attachment-actions-button--${
```
Comment (vishalnarkhede) [inline src/components/AttachmentActions.js]: prettier fix

File: src/components/ChannelListTeam.js [hunk 4]
```diff
@@ -73,9 +73,7 @@ class ChannelListTeam extends PureComponent {
                   {this.props.client.user.name || this.props.client.user.id}
                 </div>
                 <div
-                  className={`str-chat__channel-list-team__header--status ${
```
Comment (vishalnarkhede) [inline src/components/ChannelListTeam.js]: prettier fix

File: src/components/MessageCommerce.js [hunk 5]
```diff
@@ -156,9 +156,7 @@ export class MessageCommerce extends PureComponent {
         <React.Fragment>
           <span
             key={message.id}
-            className={`${messageClasses} str-chat__message--deleted ${
```
Comment (vishalnarkhede) [inline src/components/MessageCommerce.js]: prettier fix

File: src/components/MessageSimple.js [hunk 6]
```diff
@@ -358,9 +358,7 @@ export class MessageSimple extends PureComponent {
         <React.Fragment>
           <div
             key={message.id}
-            className={`${messageClasses} str-chat__message--deleted ${
```
Comment (vishalnarkhede) [inline src/components/MessageSimple.js]: prettier fix

**Comment by jaapbakker88** [comment]
Thanks for picking this up. I started on it but kind of forgot about it again. One thing to do before we can merge is styling. @vishalnarkhede do you want to do this or do you want me to have a look at it? 

image

**Pull request merged**